src/python/foglamp/core/routes.py

```diff
@@ -16,12 +16,30 @@
 def setup(app):
     # app.router.add_route('POST', '/foglamp/a-post-req', api.a_post_req, expect_handler = aiohttp.web.Request.json)
     app.router.add_route('GET', '/foglamp/ping', api.ping)
+
+    # Configuration
     app.router.add_route('GET', '/foglamp/categories', api.get_categories)
     app.router.add_route('GET', '/foglamp/category/{category_name}', api.get_category)
     app.router.add_route('GET', '/foglamp/category/{category_name}/{config_item}', api.get_category_item)
     app.router.add_route('PUT', '/foglamp/category/{category_name}/{config_item}/{value}', api.set_configuration_item)
     app.router.add_route('DELETE', '/foglamp/category/{category_name}/{config_item}', api.set_configuration_item)
 
+    # Scheduler
+    # Schedules - As per doc
+    app.router.add_route('GET', '/foglamp/schedules', api.get_schedules)
+    app.router.add_route('POST', '/foglamp/schedule', api.post_schedule)
+    app.router.add_route('GET', '/foglamp/schedule/{schedule_id}', api.get_schedule)
+    app.router.add_route('PUT', '/foglamp/schedule/{schedule_id}', api.update_schedule)
+    app.router.add_route('DELETE', '/foglamp/schedule/{schedule_id}', api.delete_schedule)
+
+    # Tasks - As per doc
+    app.router.add_route('GET', '/foglamp/tasks', api.get_tasks)
+    app.router.add_route('POST', '/foglamp/task', api.post_task)
+    app.router.add_route('GET', '/foglamp/tasks/latest', api.get_tasks_latest)
+    app.router.add_route('GET', '/foglamp/task/{task_id}', api.get_task)
+    # TODO: Find out why not DELETE/PUT a task? Cancel flag can be handled in database update.
```

Comment:
This can not be a database update, it must interact with the scheduler to make the scheduler cancel the task. The scheduler should not have to poll the database. As per PUT or DELETE, I am flexible on that, it's borderline that it should be delete as it does not remove the entity.